spec/messenger_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'messenger'
+
+describe Messenger do
+  subject(:messenger) { described_class.new }
+
+  describe '#send_message' do
+    it 'can send a message' do
+      allow(messenger).to receive(:send_message).and_return("Success")
+      message = "Your Dominos pizza is on its way!"
+      expect(messenger.send_message(message)).to eq "Success"
```

Comment:
vacuous - you are just testing the stub on line 8.

What you _actually_ want to test is that the twilio client is initialized properly (e.g. receives the #new message with the correct arguments) and that it receives the message chain messages.create (again with the correct arguments).

You could do that with a chain of doubles, each returning the next.